.palantir/revapi.yml

```diff
@@ -1056,6 +1056,36 @@ acceptedBreaks:
     - code: "java.method.removed"
       old: "method org.apache.iceberg.DataFiles.Builder org.apache.iceberg.DataFiles.Builder::withEqualityFieldIds(java.util.List<java.lang.Integer>)"
       justification: "Deprecations for 1.6.0 release"
+  "1.6.0":
+    org.apache.iceberg:iceberg-core:
+    - code: "java.method.addedToInterface"
+      new: "method <T extends org.apache.iceberg.rest.RESTResponse> T org.apache.iceberg.rest.RESTClient::delete(java.lang.String,\
+        \ java.lang.Class<T>, org.apache.iceberg.rest.auth.AuthSession, java.util.function.Consumer<org.apache.iceberg.rest.responses.ErrorResponse>)"
+      justification: "AuthManager API refactoring, no binary breaking change"
```

Comment:
it still breaks the API unfortunately. It would be better to just add a default method that throws rather than introducing API breaking changes. See also https://iceberg.apache.org/contribute/#adding-new-functionality-without-breaking-apis